sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1761,7 +1761,9 @@ class Analyzer(override val catalogManager: CatalogManager)
     try {
       innerResolve(expr, isTopLevel = true)
     } catch {
-      case _: AnalysisException if !throws => expr
+      case ae: AnalysisException if !throws =>
+        logWarning(ae.message)
```

Comment:
If you want to suggest this feature for `debug and tracing`, this should be `logDebug` or `logTrace`, @monkeyboy123 .

Reply:
Thanks for review, updated

Reply:
It seems that the unit tests errors in CI is not related to this pr

Reply:
After rebase master branch,  It is ok for CI UT, @dongjoon-hyun